dirty_cat/datasets/fetching.py

```diff
@@ -346,18 +346,23 @@ def fetch_employee_salaries():
     dict
         a dictionary containing:
 
-            - a short description of the dataset (under the ``description``
+            - a short description of the dataset (under the ``DESCR``
               key)
-            - an absolute path leading to the csv file where the data is stored
-              locally (under the ``path`` key)
+            - the tabular data (under the ``data`` key)
+            - the target (under the ``target`` key)
 
     References
     ----------
     https://catalog.data.gov/dataset/employee-salaries-2016
 
     """
 
-    return fetch_dataset(EMPLOYEE_SALARIES_CONFIG, show_progress=False)
+    from sklearn.datasets import fetch_openml
```

Comment:
I think that I would prefer if this import was moved to the top of the module, with the other imports.